forest/bonsai/simulate_gps_data.py

```diff
@@ -18,6 +18,15 @@
 from forest.poplar.legacy.common_funcs import datetime2stamp, stamp2datetime
 
 R = 6.371*10**6
+POSSIBLE_EXITS_LIST = [
```

Comment:
Would it make sense to make it an `Enum` to simplify error checking (similar to `Vehicle`)?